packages/opsian/opsian.0.1/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "2.0"
+synopsis: "Low overhead profiling"
+maintainer: "Opsian <admin@opsian.com>"
+authors:
+  "Richard Warburton <richard@opsian.com>, Sadiq Jaffer <sadiq@opsian.com>"
+license: "MIT"
+homepage: "https://github.com/Opsian/opsian-ocaml/wiki"
+bug-reports: "https://github.com/Opsian/opsian-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  ("ocaml" {>= "4.12.0" & < "5.0.0"} |
+   ("ocaml" {>= "5.0.0"} & "runtime_events_tools"))
+]
+available:
+  arch != "arm32" & arch != "x86_32" & arch != "s390x" & os-family != "arch" &
+  os-family != "alpine"
+build: ["dune" "build" "--release" "-j" jobs]
+depexts: [
+  ["libc-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["automake"] {os-family = "debian" | os-family = "ubuntu"}
+  ["libtool"] {os-family = "debian" | os-family = "ubuntu"}
+  ["liblzma-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["perl"] {os-family = "debian" | os-family = "ubuntu"}
+  ["libc-dev"] {os-family = "alpine"}
+  ["autoconf"] {os-family = "alpine"}
+  ["automake"] {os-family = "alpine"}
+  ["libtool"] {os-family = "alpine"}
+  ["xz-dev"] {os-family = "alpine"}
+  ["automake"] {os-family = "arch"}
+  ["libtool"] {os-family = "arch"}
+  ["xz"] {os-family = "arch"}
+  ["glibc-devel"]
+    {os-family = "fedora" | os-distribution = "rhel" | os-family = "centos"}
+  ["automake"]
+    {os-family = "fedora" | os-distribution = "rhel" | os-family = "centos"}
+  ["libtool"]
+    {os-family = "fedora" | os-distribution = "rhel" | os-family = "centos"}
+  ["xz-devel"]
+    {os-family = "fedora" | os-distribution = "rhel" | os-family = "centos"}
+  ["perl-core"]
+    {os-family = "fedora" | os-distribution = "rhel" | os-family = "centos"}
+  ["glibc-devel"] {os-family = "suse" | os-family = "opensuse"}
+  ["automake"] {os-family = "suse" | os-family = "opensuse"}
+  ["libtool"] {os-family = "suse" | os-family = "opensuse"}
+  ["xz-devel"] {os-family = "suse" | os-family = "opensuse"}
+  ["perl-core"]
+    {os-family = "fedora" | os-distribution = "rhel" | os-family = "centos"}
+]
+dev-repo: "git://git@github.com/Opsian/opsian-ocaml.git"
+url {
+  src:
+    "https://github.com/Opsian/opsian-ocaml/releases/download/0.1/0.1.tar.gz"
```

Comment:
Note for next time that this archive contains the whole `.git` directory which shouldn’t be required for building and makes the archive bigger than it needs to

Reply:
Noted and understood. We've been using git submodules to manage dependencies here. The only other option appears to be committing the sources of those dependencies into the repository. Is that preferred?

Reply:
why is that? The archive already contains all the submodules. Unless you somehow use `git` during the compilation process (which shouldn’t be needed), there is no need for the git directory to be present in that particular archive